cmd/init.go

```diff
@@ -260,6 +268,20 @@ func validatePrivateTransactionManagerBlockchainConnectorCombination(privateTran
 	return nil
 }
 
+func validateRemoteNodeVsBlockChainNodeFlags(remoteNodeUrl, blockchainNodePorvider string) error {
+	if len(remoteNodeUrl) != 0 && blockchainNodePorvider != "geth" {
+		return errors.New("`remote-noted-url` and `blockchain-node` can't both be specified")
```

Comment:
```suggestion
		return errors.New("Both flags `remote-node-url` and `blockchain-node` can't be specified")
```

Reply:
hi @EnriqueL8, thanks for your review. I will correct the text and look for a place where to place the test.

Reply:
hi @EnriqueL8, now that I took a deeper look, I remembered why I didn't provide tests initially. I didn't find any of the `initCommon` behavior to have been tested. Can you please let me know if I'm mistaken, or if not - and still a test needs to be written for this part - where, according to the practices of this repo, shall it be?

Thanks in advance for your help.

Reply:
okay this might more significant work to establish a test framework for these